src/compiler/parser.ts

```diff
@@ -2054,16 +2054,17 @@ namespace ts {
             parsingContext |= 1 << kind;
             const list = [];
             const listPos = getNodePos();
+            let lastElement: Node | undefined;

```

Comment:
@CyrusNajmabadi you know about incremental parsing. I can't imagine that there are any issues here since the parser still runs through all the list elements correctly. Do you have any thoughts on this strategy?